TUM Campus App/CardViewController.swift

```diff
@@ -72,21 +75,40 @@ extension CardViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         setupLogo()
+        
+        if #available(iOS 11.0, *) {
+            self.navigationController?.navigationBar.prefersLargeTitles = false
+            self.navigationController?.navigationItem.largeTitleDisplayMode = .never
+        }
 
         refresh.addTarget(self, action: #selector(CardViewController.refresh(_:)), for: UIControlEvents.valueChanged)
         tableView.addSubview(refresh)
         tableView.tableFooterView = UIView(frame: CGRect.zero)
         tableView.separatorStyle = .none
         tableView.backgroundColor = .white
-        manager = (self.tabBarController as? CampusTabBarController)?.manager
+        definesPresentationContext = true
+
+        let storyboard = UIStoryboard(name: "CardView", bundle: nil)
+        guard let searchResultsController = storyboard.instantiateViewController(withIdentifier: "SearchResultsController") as? SearchResultsController else {
+            fatalError("Unable to instatiate a SearchResultsViewController from the storyboard.")
+        }
+        search = UISearchController(searchResultsController: searchResultsController)
+        search?.searchResultsUpdater = self
+        search?.searchBar.placeholder = "Search"
+        search?.obscuresBackgroundDuringPresentation = true
+        search?.hidesNavigationBarDuringPresentation = true
+        if #available(iOS 11.0, *) {
+            self.navigationItem.searchController = search
+        }
```

Comment:
People with iOS 10 should probably still be able to search. 
We should default to an extra search view controller here.